scripts/postcss/ThemePlugin.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs-extra');
+const path = require('path');
+const postcss = require('postcss');
+const removeCssModulesPseudoClasses = require('./_removeCssModulesPseudoClasses');
+
+const CONFIG = 'terra-theme.config.js';
+const SUPPORTED_THEMES = [
+  'orion-fusion-theme',
+  'clinical-lowlight-theme',
+  'cerner-clinical-theme',
+];
+
+/**
+ * The purpose of this plugin is to create a default theme from a scoped theme
+ * and to remove any supported themes that are not desired.
+ */
+module.exports = postcss.plugin('postcss-test-plugin', (config) => {
+  // Retrieve theme config.
+  let themeConfig = {};
+  if (config) {
+    themeConfig = config;
+  } else {
+    const defaultConfig = path.resolve(process.cwd(), CONFIG);
+    if (fs.existsSync(defaultConfig)) {
+      // eslint-disable-next-line global-require, import/no-dynamic-require
+      themeConfig = require(defaultConfig);
+    }
+  }
+
+  // Add the . to find the selector.
+  const defaultThemeSelector = `.${themeConfig.theme}`;
+
+  const scopedThemes = themeConfig.scoped || [];
+
+  // Find the set of known themes that should not be included.
+  const themesToRemove = SUPPORTED_THEMES.reduce((acc, theme) => {
```

Comment:
For whatever themes should be included after this step, should they be looped to be transformed from `theme` to `.theme`?

Reply:
I'm confused, we transform themes to remove below and we don't include any more themes after this step?

Reply:
I guess I'm assuming the scope list could include a theme not included in our supported list. My thought was they were defined as:
```
scope: [
    'custom-theme',
    'orion-fusion-theme',
]
```
Maybe I'm reading this wrong, but this is filter out the themes right so the themesToRemove value is:
```
themesToRemove = [
    '.clinical-lowlight-theme',
    '.cerner-clinical-theme'
]
```
Won't the remaining scoped themes need to be updated to be:
```
themesToInclude = [
    '.custom-theme',
    '.orion-fusion-theme',
]
```

Reply:
Actually, we only care about the ones to remove, we aren't dong anything special to pull in the scoped themes because we expect that to happen in the scss file via import. Whoops. I figured it out 👍